packages/core/test/foundry/optimistic-assertor/OptimisticAsserter.Edgecases.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "../fixtures/optimistic-assertor/OptimisticAssertorFixture.sol";
+import "../fixtures/common/TestAddress.sol";
+
+contract Boundaries is Test {
+    OptimisticAssertor optimisticAssertor;
+    TestnetERC20 defaultCurrency;
+    Timer timer;
+    string claimAssertion = 'q:"The sky is blue"';
+
+    function setUp() public {
+        OptimisticAssertorFixture.OptimisticAsserterContracts memory oaContracts =
+            new OptimisticAssertorFixture().setUp();
+        optimisticAssertor = oaContracts.optimisticAssertor;
+        defaultCurrency = oaContracts.defaultCurrency;
+        timer = oaContracts.timer;
+    }
+
+    function test_RevertIf_BondBelowMinimum() public {
```

Comment:
I'm trying to follow the [foundry general-test-guidance](https://book.getfoundry.sh/tutorials/best-practices?highlight=get%20event%20topics#general-test-guidance)